<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.553">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Pingjian Ding - Publications</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Pingjian Ding</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./researches.html"> 
<span class="menu-text">Researches</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./publications.html" aria-current="page"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./courses.html"> 
<span class="menu-text">Courses</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./blogs.html"> 
<span class="menu-text">Blogs</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./contact.html"> 
<span class="menu-text">Contact</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#section" id="toc-section" class="nav-link active" data-scroll-target="#section">2024</a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1">2023</a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2">2022</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3">2021</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4">2020</a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5">2019</a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6">2018</a></li>
  <li><a href="#section-7" id="toc-section-7" class="nav-link" data-scroll-target="#section-7">2017</a></li>
  <li><a href="#section-8" id="toc-section-8" class="nav-link" data-scroll-target="#section-8">2016</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="section" class="level3">
<h3 class="anchored" data-anchor-id="section">2024</h3>
<ul>
<li><p>Zhenxiang Gao#, <strong>Pingjian Ding</strong>#, and Rong Xu*. “IUPHAR review-Data-driven Computational Drug Repurposing Approaches for Opioid Use Disorder.” Pharmacological Research 199 (2024): 106960. <a href="https://www.sciencedirect.com/science/article/pii/S104366182300316X">[Link]</a></p></li>
<li><p>Hanyu Luo, Li Tang, Min Zeng, Rui Yin, <strong>Pingjian Ding</strong>, Lingyun Luo, and Min Li. “BertSNR: an interpretable deep learning framework for single nucleotide resolution identification of transcription factor binding sites based on DNA language model.” Bioinformatics (2024). <a href="https://academic.oup.com/bioinformatics/article/40/8/btae461/7728457">[Link]</a></p></li>
<li><p>Anlin Hou, Hanyu Luo, Huan Liu, Lingyun Luo, and <strong>Pingjian Ding</strong>. “Multi-scale DNA language model improves 6 mA binding sites prediction.” Computational Biology and Chemistry 112(17):108129. (2024). <a href="https://www.sciencedirect.com/science/article/abs/pii/S1476927124001178">[Link]</a></p></li>
<li><p>Zhenyu Jiang, <strong>Pingjian Ding</strong>, Cong Shen, and Xiaopeng Dai. “Geometric molecular graph representation learning model for drug-drug interactions prediction.” (2024). IEEE Journal of Biomedical and Health Informatics. <a href="https://ieeexplore.ieee.org/document/10663713">[Link]</a></p></li>
<li><p>Yuxun Luo, Wenyu Shan, Li Peng, Lingyun Luo, <strong>Pingjian Ding</strong>, and Wei Liang. “A computational framework for predicting novel drug indications using graph convolutional network with contrastive learning.” IEEE Journal of Biomedical and Health Informatics (2024). <a href="https://ieeexplore.ieee.org/document/10497844">[Link]</a></p></li>
<li><p>Ziyu Wu, Shasha Li, Lingyun Luo, and <strong>Pingjian Ding*</strong>. “HKFGCN: A novel multiple kernel fusion framework on graph convolutional network to predict microbe-drug associations.” Computational Biology and Chemistry (2024): 108041. <a href="https://www.sciencedirect.com/science/article/abs/pii/S147692712400029X">[Link]</a></p></li>
<li><p>Yuxun Luo, Shasha Li, Li Peng, <strong>Pingjian Ding</strong>, and Wei Liang. “Predicting associations between drugs and G Protein-Coupled Receptors using a multi-graph convolutional network.” Computational Biology and Chemistry (2024): 108060. <a href="https://www.sciencedirect.com/science/article/abs/pii/S1476927124000483">[Link]</a></p></li>
<li><p>Cong Shen, <strong>Pingjian Ding</strong>, Junjie Wee, Jialin Bi, Jiawei Luo, and Kelin Xiaa. “Curvature-enhanced graph convolutional network for biomolecular interaction prediction.” Computational and Structural Biotechnology Journal (2024). <a href="https://www.sciencedirect.com/science/article/pii/S2001037024000291">[Link]</a></p></li>
<li><p>Min Li, Baoying Zhao, Yiming Li, <strong>Pingjian Ding</strong>, Rui Yin, Shichao Kan, Yi Pan, and Min Zeng. “SGCL-LncLoc: an interpretable deep learning model for improving lncRNA subcellular localization prediction with supervised graph contrastive learning.” Big Data Mining and Analytics (2024). <a href="https://ieeexplore.ieee.org/document/10654677">[Link]</a></p></li>
<li><p>Jiang, Zhenyu, Zhi Gong, Xiaopeng Dai, Hongyan Zhang, <strong>Pingjian Ding</strong>, and Cong Shen. “Deep graph contrastive learning model for drug-drug interaction prediction.” PloS one 19, no. 6 (2024): e0304798. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0304798">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-1" class="level3">
<h3 class="anchored" data-anchor-id="section-1">2023</h3>
<ul>
<li><p>Pan Zheng*, Xiangxiang Zeng, Xun Wang, <strong>Pingjian Ding</strong>. “Editorial: Artificial intelligence and machine learning for drug discovery, design and repurposing: methods and applications.” Frontiers in Pharmacology (2023). <a href="https://www.frontiersin.org/journals/pharmacology/articles/10.3389/fphar.2023.1333747/full">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, and Rong Xu*. “Causal association of COVID-19 with brain structure changes: Findings from a non-overlapping 2-sample Mendelian randomization study.” Journal of the Neurological Sciences (2023): 2023-07. <a href="https://www.sciencedirect.com/science/article/pii/S0022510X23003258">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Mark Gurney, George Perry, Rong Xu*. “Association of COVID-19 with risk and progression of Alzheimer’s disease: non-overlapping two-sample Mendelian randomization analysis of 2.6 million subjects.” Journal of Alzheimer’s Disease (2023). <a href="https://content.iospress.com/articles/journal-of-alzheimers-disease/jad230632">[Link]</a></p></li>
<li><p>Wenyu Shan, Cong Shen, Lingyun Luo, and <strong>Pingjian Ding</strong>*. “Multi-task Learning for Predicting Synergistic Drug Combinations based on Auto-Encoding Multi-Relational Graphs.” iScience (2023). <a href="https://www.sciencedirect.com/science/article/pii/S2589004223020977">[Link]</a></p></li>
<li><p>Yichen Zhong, Xiaoting Xi, Cong Shen, Yuxun Luo, <strong>Pingjian Ding</strong>, and Lingyun Luo*. “Multitask joint learning with graph autoencoders for predicting potential MiRNA-drug associations.” Artificial Intelligence in Medicine (2023): 102665. <a href="https://www.sciencedirect.com/science/article/pii/S0933365723001793?casa_token=aa0DI53VXO4AAAAA:FUUbvoBanXwVLwAUk2t74FuQbR4xDBcMjntLcz5UN-Q-5UMcnctjA8YVSorbUfopr5lELS2NKQ">[Link]</a></p></li>
<li><p>Cheng Chen, Lingyun Luo*, Chunlei Zheng, <strong>Pingjian Ding</strong>*, Huan Liu, and Hanyu Luo*. “Self-prediction of relations in GO facilitates its quality auditing.” Journal of Biomedical Informatics 144 (2023): 104441. <a href="https://www.sciencedirect.com/science/article/pii/S1532046423001624?casa_token=iIvwTXG3kvUAAAAA:eDwlxIjPvprByu74MQ-5IGtSpWEnagndI1Mhhzb1aKgO8ip3jgLclv1JNoIy9omIjpfZqbY9QQ">[Link]</a></p></li>
<li><p>Hanyu Luo, Ye Li, Huan Liu, <strong>Pingjian Ding</strong>, Ying Yu, and Lingyun Luo*. “SENet: a deep learning framework for discriminating super-and typical enhancers by sequence information.” Computational Biology and Chemistry (2023): 107905. <a href="https://www.sciencedirect.com/science/article/pii/S1476927123000968?casa_token=tLlIHDguSm8AAAAA:e7Lrz0e6cqWFHlwJNP2LxcGathtLhmqz-Nxpo9RWCQgUEOjNpFgT7697GP8YQjC_UBbwDlwqMg">[Link]</a></p></li>
<li><p>Hanyu Luo, Wenyu Shan, Cheng Chen, <strong>Pingjian Ding</strong>, and Lingyun Luo*. “Improving language model of human genome for DNA–protein binding prediction based on task-specific pre-training.” Interdisciplinary Sciences: Computational Life Sciences 15, no. 1 (2023): 32-43. <a href="https://link.springer.com/article/10.1007/s12539-022-00537-9">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>*, Min Zeng*, and Rui Yin*. “Editorial: Computational methods to analyze RNA data for human diseases.” Frontiers in Genetics 14 (2023). <a href="https://www.frontiersin.org/articles/10.3389/fgene.2023.1270334/full">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Maria P. Gorenflo, Xiaofeng Zhu, and Rong Xu*. “Aspirin Use and Risk of Alzheimer’s Disease: A 2-Sample Mendelian Randomization Study.” Journal of Alzheimer’s Disease (2023): 1-12. <a href="https://content.iospress.com/articles/journal-of-alzheimers-disease/jad220787">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-2" class="level3">
<h3 class="anchored" data-anchor-id="section-2">2022</h3>
<ul>
<li><p><strong>Pingjian Ding</strong>#, Yiheng Pan#, Quanqiu Wang, and Rong Xu*. “Prediction and evaluation of combination pharmacotherapy using natural language processing, machine learning and patient electronic health records.” Journal of Biomedical Informatics 133 (2022): 104164. <a href="https://www.sciencedirect.com/science/article/pii/S1532046422001769">[Link]</a></p></li>
<li><p>Hanyu Luo, Cheng Chen, Wenyu Shan, <strong>Pingjian Ding</strong>, and Lingyun Luo*. “iEnhancer-BERT: A novel transfer learning architecture based on DNA-Language model for identifying enhancers and their strength.” In International Conference on Intelligent Computing, pp.&nbsp;153-165. Cham: Springer International Publishing, 2022. <a href="https://link.springer.com/chapter/10.1007/978-3-031-13829-4_13">[Link]</a></p></li>
<li><p>Zhenxiang Gao, <strong>Pingjian Ding</strong>, and Rong Xu*. “KG-Predict: A knowledge graph computational framework for drug repurposing.” Journal of biomedical informatics 132 (2022): 104133. <a href="https://www.sciencedirect.com/science/article/pii/S1532046422001496">[Link]</a></p></li>
<li><p>Zhenxiang Gao, Yiheng Pan, <strong>Pingjian Ding</strong>, and Rong Xu*. “A knowledge graph-based disease-gene prediction system using multi-relational graph convolution networks.” In AMIA Annual Symposium Proceedings, vol.&nbsp;2022, p.&nbsp;468. American Medical Informatics Association, 2022. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10148306/">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-3" class="level3">
<h3 class="anchored" data-anchor-id="section-3">2021</h3>
<ul>
<li><p>Xin Chen, Lingyun Luo, Cong Shen, <strong>Pingjian Ding</strong>*, and Jiawei Luo. “An in silico method for predicting drug synergy based on multitask learning.” Interdisciplinary Sciences: Computational Life Sciences 13 (2021): 299-311. <a href="https://link.springer.com/article/10.1007/s12539-021-00422-x">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Cheng Liang, Wenjue Ouyang, Guanghui Li, Qiu Xiao, and Jiawei Luo*. “Inferring synergistic drug combinations based on symmetric meta-path in a novel heterogeneous network.” IEEE/ACM Transactions on Computational Biology and Bioinformatics 18, no. 4 (2019): 1562-1571. <a href="https://ieeexplore.ieee.org/abstract/document/8892647?casa_token=6iZi9CAZOcoAAAAA:kQrhY-7ZZ3Zjm-iYib2ggJfdQ_DcWRfkOQQGPjynGtsTtW9c3DmTGV0SVzbcJS4v8ES1e9tC">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-4" class="level3">
<h3 class="anchored" data-anchor-id="section-4">2020</h3>
<ul>
<li><p>Cong Shen, Jiawei Luo*, Wenjue Ouyang, <strong>Pingjian Ding</strong>, and Xiangtao Chen. “IDDkin: network-based influence deep diffusion model for enhancing prediction of kinase inhibitors.” Bioinformatics 36, no. 22-23 (2020): 5481-5491. <a href="https://academic.oup.com/bioinformatics/article/36/22-23/5481/6050789">[Link]</a></p></li>
<li><p>Li, Guanghui, Jiawei Luo*, Diancheng Wang, Cheng Liang, Qiu Xiao, <strong>Pingjian Ding</strong>, and Hailin Chen. “Potential circRNA-disease association prediction using DeepWalk and network consistency projection.” Journal of biomedical informatics 112 (2020): 103624. <a href="https://www.sciencedirect.com/science/article/pii/S1532046420302525">[Link]</a></p></li>
<li><p>Cong Shen, Jiawei Luo*, Wenjue Ouyang, <strong>Pingjian Ding</strong>, and Hao Wu. “Identification of small molecule–miRNA associations with graph regularization techniques in heterogeneous networks.” Journal of Chemical Information and Modeling 60, no. 12 (2020): 6709-6721. <a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.0c00975?casa_token=dJLeU4ps-S8AAAAA%3AFVDspi-cUt4bEmdAtXNcQim7-mO2Gl7OPSMO0R5Nezyt2cVJyxjSISaVdNNQG9Vc6ktYsaBaN7SByc0">[Link]</a></p></li>
<li><p>Qiu Xiao, Haiming Yu, Jiancheng Zhong, Cheng Liang, Guanghui Li, <strong>Pingjian Ding</strong>, and Jiawei Luo*. “An in-silico method with graph-based multi-label learning for large-scale prediction of circRNA-disease associations.” Genomics 112, no. 5 (2020): 3407-3415. <a href="https://www.sciencedirect.com/science/article/pii/S088875432030505X">[Link]</a></p></li>
<li><p>Cong Shen, Jiawei Luo, Zihan Lai, and <strong>Pingjian Ding</strong>. “Multiview joint learning-based method for identifying small-molecule-associated MiRNAs by integrating pharmacological, genomics, and network knowledge.” Journal of Chemical Information and Modeling 60, no. 8 (2020): 4085-4097. <a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.0c00244?casa_token=ibcensq9K40AAAAA%3AwXCwihiGtwCfrEKQgL5YtEU3G_he24up9Cgu_e9Oi1e8kMSIee3hihjvh6cxg7iOh8am-QNpG3hgjjk">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Wenjue Ouyang, Jiawei Luo*, and Chee-Keong Kwoh. “Heterogeneous information network and its application to human health and disease.” Briefings in bioinformatics 21, no. 4 (2020): 1327-1346. <a href="https://academic.oup.com/bib/article/21/4/1327/5574891">[Link]</a></p></li>
<li><p>Jiawei Luo*, Cong Shen, Zihan Lai, Jie Cai, and <strong>Pingjian Ding</strong>. “Incorporating clinical, chemical and biological information for predicting small molecule-microRNA associations based on non-negative matrix factorization.” IEEE/ACM Transactions on Computational Biology and Bioinformatics 18, no. 6 (2020): 2535-2545. <a href="https://ieeexplore.ieee.org/abstract/document/9007520?casa_token=ERyWkP7VsAkAAAAA:VM7E1Bq-v5ZPVANXhb5axiSiqp612pWPrOT-kyxgV7VHfzGGhohqUQN3MJU9Kt3YyDKNjM42">[Link]</a></p></li>
<li><p>Haojiang Tan, Quanmeng Sun, Guanghui Li*, Qiu Xiao, <strong>Pingjian Ding</strong>, Jiawei Luo, and Cheng Liang. “Multiview consensus graph learning for lncRNA–disease association prediction.” Frontiers in genetics 11 (2020): 89. <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.00089/full">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Cong Shen, Zihan Lai, Cheng Liang, Guanghui Li, and Jiawei Luo*. “Incorporating multisource knowledge to predict drug synergy based on graph co-regularization.” Journal of Chemical Information and Modeling 60, no. 1 (2019): 37-46. <a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.9b00793?casa_token=WYgfr2QK450AAAAA%3Az80IfjECL4LYT8vcu7Mx-X8He-E5XU7MWCqd65u5kRoMFtfea0kYYc8wIPdmwiS9JdcO3CtYX0Alx_4">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-5" class="level3">
<h3 class="anchored" data-anchor-id="section-5">2019</h3>
<ul>
<li><p>Zhenxia Pan, Huaxiang Zhang, Cheng Liang*, Guanghui Li, Qiu Xiao, <strong>Pingjian Ding</strong>, and Jiawei Luo. “Self-weighted multi-kernel multi-label learning for potential miRNA-disease association prediction.” Molecular Therapy-Nucleic Acids 17 (2019): 414-423. <a href="https://www.cell.com/molecular-therapy-family/nucleic-acids/fulltext/S2162-2531(19)30177-5">[Link]</a></p></li>
<li><p>Guanghui Li*, Jiawei Luo, Cheng Liang, Qiu Xiao, <strong>Pingjian Ding</strong>, and Yuejin Zhang. “Prediction of LncRNA-disease associations based on network consistency projection.” Ieee Access 7 (2019): 58849-58856. <a href="https://ieeexplore.ieee.org/abstract/document/8705296">[Link]</a></p></li>
<li><p>Shengpeng Yu, Cheng Liang*, Qiu Xiao, Guanghui Li, <strong>Pingjian Ding</strong>, and JiaWei Luo. “MCLPMDA: A novel method for mi RNA‐disease association prediction based on matrix completion and label propagation.” Journal of cellular and molecular medicine 23, no. 2 (2019): 1427-1438. <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/jcmm.14048">[Link]</a></p></li>
<li><p>Guanghui Li*, Yingjie Yue, Cheng Liang, Qiu Xiao, <strong>Pingjian Ding</strong>, and Jiawei Luo. “NCPCDA: network consistency projection for circRNA–disease association prediction.” RSC advances 9, no. 57 (2019): 33222-33228. <a href="https://pubs.rsc.org/en/content/articlehtml/2019/ra/c9ra06133a">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Rui Yin, Jiawei Luo*, and Chee-Keong Kwoh. “Ensemble prediction of synergistic drug combinations incorporating biological, chemical, pharmacological, and network knowledge.” IEEE journal of biomedical and health informatics 23, no. 3 (2019): 1336-1345.<a href="https://ieeexplore.ieee.org/abstract/document/8401869?casa_token=VkduwP9Cs2UAAAAA:xNeLq4GxEZhnfOvZhjj42gVxR3O-uU0q1L0ipd1Gco6UaUZWQIAdXWCiWv7f_clQ6s7-NYpn">[Link]</a></p></li>
<li><p>Ying Liu, Jiawei Luo*, and <strong>Pingjian Ding</strong>. “Inferring microRNA targets based on restricted Boltzmann machines.” IEEE journal of biomedical and health informatics 23, no. 1 (2019): 427-436.<a href="https://ieeexplore.ieee.org/abstract/document/8310896?casa_token=kD9lWS1l_2gAAAAA:YT8Bj6yFhgXsMeT_SuU0pSoIfcgKoP-TBShit2DPfEVOABZI_FgQFAJL4qRveKqSKW8iIJtz">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-6" class="level3">
<h3 class="anchored" data-anchor-id="section-6">2018</h3>
<ul>
<li><p>Shengpeng Yu, Cheng Liang*, Qiu Xiao, Guanghui Li, <strong>Pingjian Ding</strong>, and Jiawei Luo. “GLNMDA: a novel method for miRNA-disease association prediction based on global linear neighborhoods.” RNA biology 15, no. 9 (2018): 1215-1227. <a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2018.1521210">[Link]</a></p></li>
<li><p>Xiao, Qiu, Jiawei Luo*, Cheng Liang, Guanghui Li, Jie Cai, <strong>Pingjian Ding</strong>, and Ying Liu. “Identifying lncRNA and mRNA co-expression modules from matched expression data in ovarian cancer.” IEEE/ACM transactions on computational biology and bioinformatics 17, no. 2 (2018): 623-634. <a href="https://ieeexplore.ieee.org/abstract/document/8428478?casa_token=YbjXys6DQiwAAAAA:Zqw3Ssf5hk5KkggNc9RXKShzVNCzrizmZiWD8wIeg6RJNvavuj0nmuA0vVmx8FOvjxbEEP0R">[Link]</a></p></li>
<li><p>Yu Qu, Huaxiang Zhang, Cheng Liang*, <strong>Pingjian Ding</strong>, and Jiawei Luo. “SNMDA: A novel method for predicting micro RNA‐disease associations based on sparse neighbourhood.” Journal of Cellular and Molecular Medicine 22, no. 10 (2018): 5109-5120. <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/jcmm.13799">[Link]</a></p></li>
<li><p>Buwen Cao*, Shuguang Deng, Hua Qin, <strong>Pingjian Ding</strong>, Shaopeng Chen, and Guanghui Li. “Detection of protein complexes based on penalized matrix decomposition in a sparse protein–protein interaction network.” Molecules 23, no. 6 (2018): 1460. <a href="https://www.mdpi.com/1420-3049/23/6/1460">[Link]</a></p></li>
<li><p>Jiawei Luo*, <strong>Pingjian Ding</strong>, Cheng Liang, and Xiangtao Chen. “Semi-supervised prediction of human miRNA-disease association based on graph regularization framework in heterogeneous networks.” Neurocomputing 294 (2018): 29-38. <a href="https://www.sciencedirect.com/science/article/pii/S0925231218302674?casa_token=o6MfG_leyN4AAAAA:goW74Fp9lzrhxuCe1LktCX8CHgVJXrR8tTMUlSCLNwMHkVIwVMHy3r-KeEDf0FJvpSiZsK9EYw">[Link]</a></p></li>
<li><p>Guanghui Li*, Jiawei Luo, Qiu Xiao, Cheng Liang, and <strong>Pingjian Ding</strong>. “Predicting microRNA-disease associations using label propagation based on linear neighborhood similarity.” Journal of biomedical informatics 82 (2018): 169-177. <a href="https://www.sciencedirect.com/science/article/pii/S1532046418300881">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Jiawei Luo*, Cheng Liang, Qiu Xiao, Buwen Cao, and Guanghui Li. “Discovering synergistic drug combination from a computational perspective.” Current Topics in Medicinal Chemistry 18, no. 12 (2018): 965-974. <a href="https://www.ingentaconnect.com/content/ben/ctmc/2018/00000018/00000012/art00004">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Jiawei Luo*, Cheng Liang, Qiu Xiao, and Buwen Cao. “Human disease MiRNA inference by combining target information based on heterogeneous manifolds.” Journal of biomedical informatics 80 (2018): 26-36. <a href="https://www.sciencedirect.com/science/article/pii/S1532046418300327">[Link]</a></p></li>
<li><p>Qiu Xiao, Jiawei Luo*, Cheng Liang, Jie Cai, and <strong>Pingjian Ding</strong>. “A graph regularized non-negative matrix factorization method for identifying microRNA-disease associations.” Bioinformatics 34, no. 2 (2018): 239-248. <a href="https://academic.oup.com/bioinformatics/article/34/2/239/4101940">[Link]</a></p></li>
<li><p>Qiao Zhu, Jiawei Luo*, <strong>Pingjian Ding</strong>, and Qiu Xiao. “GRTR: Drug-disease association prediction based on graph regularized transductive regression on heterogeneous network.” In Bioinformatics Research and Applications: 14th International Symposium, ISBRA 2018, Beijing, China, June 8-11, 2018, Proceedings 14, pp.&nbsp;13-25. Springer International Publishing, 2018. <a href="https://link.springer.com/chapter/10.1007/978-3-319-94968-0_2">[Link]</a></p></li>
<li><p>Guanghui Li, Jiawei Luo*, Qiu Xiao, Cheng Liang, and <strong>Pingjian Ding</strong>. “Prediction of microRNA–disease associations with a Kronecker kernel matrix dimension reduction model.” RSC advances 8, no. 8 (2018): 4377-4385. <a href="https://pubs.rsc.org/en/content/articlehtml/2018/ra/c7ra12491k">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-7" class="level3">
<h3 class="anchored" data-anchor-id="section-7">2017</h3>
<ul>
<li><p>Guanghui Li, Jiawei Luo*, Qiu Xiao, Cheng Liang, <strong>Pingjian Ding</strong>, and Buwen Cao. “Predicting microrna-disease associations using network topological similarity based on deepwalk.” Ieee Access 5 (2017): 24032-24039. <a href="https://ieeexplore.ieee.org/abstract/document/8085107">[Link]</a></p></li>
<li><p>Jiawei Luo*, Qiu Xiao, Cheng Liang, and <strong>Pingjian Ding</strong>. “Predicting MicroRNA-disease associations using Kronecker regularized least squares based on heterogeneous omics data.” Ieee Access 5 (2017): 2503-2513. <a href="https://ieeexplore.ieee.org/abstract/document/7862218">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Jiawei Luo*, Cheng Liang, Jie Cai, Ying Liu, and Xiangtao Chen. “A novel group wise-based method for calculating human miRNA functional similarity.” IEEE Access 5 (2017): 2364-2372. <a href="https://ieeexplore.ieee.org/abstract/document/7819459">[Link]</a></p></li>
<li><p>Buwen Cao, Shuguang Deng, Jiawei Luo*, <strong>Pingjian Ding</strong>, and Shulin Wang. “Identification of overlapping protein complexes by fuzzy K-medoids clustering algorithm in yeast protein-protein interaction networks.” Journal of Intelligent &amp; Fuzzy Systems 34, no. 1 (2018): 93-103. <a href="https://content.iospress.com/articles/journal-of-intelligent-and-fuzzy-systems/ifs17026">[Link]</a></p></li>
<li><p>Jiawei Luo*, <strong>Pingjian Ding</strong>, Cheng Liang, Buwen Cao, and Xiangtao Chen. “Collective prediction of disease-associated miRNAs based on transduction learning.” IEEE/ACM transactions on computational biology and bioinformatics 14, no. 6 (2017): 1468-1475. <a href="https://ieeexplore.ieee.org/abstract/document/7542592?casa_token=Dg-H-9yoMeYAAAAA:fJo994tl1KgqQ-J2ZcgDx9LE6wELLnEhludITGe-hLJaQ7AUgQwTNHuEdPoUZwQKt2yZoG2o">[Link]</a></p></li>
</ul>
<hr>
</section>
<section id="section-8" class="level3">
<h3 class="anchored" data-anchor-id="section-8">2016</h3>
<ul>
<li><p>Buwen Cao, Jiawei Luo*, Cheng Liang, Shulin Wang, and <strong>Pingjian Ding</strong>. “Pce-fr: A novel method for identifying overlapping protein complexes in weighted protein-protein interaction networks using pseudo-clique extension based on fuzzy relation.” IEEE transactions on nanobioscience 15, no. 7 (2016): 728-738. <a href="https://ieeexplore.ieee.org/abstract/document/7572125?casa_token=D1NOBzZvJtIAAAAA:RrJfDkFlgIZ5B0knB8AqcYj9P0l3W2OM0R6pogTaR2OKhTJRodMpCOfT96jE1rIZgMbUKQV7">[Link]</a></p></li>
<li><p><strong>Pingjian Ding</strong>, Jiawei Luo*, Qiu Xiao, and Xiangtao Chen. “A path-based measurement for human miRNA functional similarities using miRNA-disease associations.” Scientific Reports 6, no. 1 (2016): 32533. <a href="https://www.nature.com/articles/srep32533">[Link]</a></p></li>
<li><p>Jiawei Luo*, Cong Huang, and <strong>Pingjian Ding</strong>. “A meta-path-based prediction method for human miRNA-target association.” BioMed Research International 2016 (2016). <a href="https://www.hindawi.com/journals/bmri/2016/7460740/">[Link]</a></p></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Copyright 2023, Pingjian Ding</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>